test/float8/test_compile.py

```diff
@@ -318,8 +327,9 @@ def __exit__(self, *args):
         sys.stderr = self.sys_stderr
 
 
+@unittest.skipIf(not torch.accelerator.is_available(), "GPU not available")
 @unittest.skipIf(
-    not is_sm_at_least_89(),
+    torch.cuda.is_available() and not is_sm_at_least_89(),
     "CUDA not available",
 )
```

Comment:
this condition is cuda only, does it apply to XPU as well?

Reply:
We just want enable this UT on XPU and not break the condition of CUDA.